web/src/js/__tests__/components/ContentView/ContentLoaderSpec.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import ContentLoaderGenerator from '../../../components/ContentView/ContentLoader'
+import { TFlow } from '../../ducks/tutils'
+import TestUtils from 'react-dom/test-utils'
+import mockXMLHttpRequest from 'mock-xmlhttprequest'
+
+global.XMLHttpRequest = mockXMLHttpRequest
```

Comment:
If we don't mock `XMLHttpRequest`, it fails like this:
![default](https://user-images.githubusercontent.com/9366279/27015783-d5594a0a-4f49-11e7-8103-92c18f515596.png)